HeadersImports.md

```diff
@@ -86,3 +86,10 @@ inconsistent with expectations of C++ developers. "Private" headers are availabl
 via an explicit import. "Internal" or "Project" headers are only available to their enclosing
 library. Many file names in this repo include "Private" or "Internal" do not comply. Always
 check the build definition to see how the file is used.
+
+### Public Header Location Explanation
+
+CocoaPods flattens all headers specified in the podspec and makes them available via
+`#import "FirebaseFoo/Header.h"`. Swift Package Manager does not flatten. Therefore, the
+directory structure described above allows clients to import headers from either package manager
+with `#if` checks.
```

Comment:
Could you please provide a bit more details what `#if` checks are meant.

Reply:
Good catch! It was backwards. Is it clear as `without`?

Reply:
yep, looks great now!